core/modules/filters/kin.js

```diff
@@ -0,0 +1,107 @@
+/*\
+title: $:/core/modules/filters/kin.js
+type: application/javascript
+module-type: filteroperator
+
+Filter operator that recursively finds kindred between tiddlers
+
+\*/
+(function () {
+
+  /*jslint node: true, browser: true */
+  /*global $tw: false */
+  "use strict";
+
+  // TODO: Should I set global tw to true?
```

Comment:
Yes, I think it should be true here (I notice that quite a few source files set it wrongly)